server/db/postgres/sequelize_config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  development: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_development',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  test: {
+    username: process.env.PGUSER || 'root',
+    password: null,
+    database: 'react_webpack_node_test',
+    host: '127.0.0.1',
+    dialect: 'postgres'
+  },
+  production: {
+    use_env_variable: 'POSTGRES_DB_URL'
+  }
```

Comment:
Now it uses the env variable in prod but falls back to similar settings to dev, so as long as you don't have the env variable set locally to something you didn't want and you have the prod database created, everything will work just like it did in dev